zio-http/src/main/scala/zio/http/ServerConfig.scala

```diff
@@ -0,0 +1,182 @@
+package zio.http
+
+import io.netty.bootstrap.ServerBootstrap
+import io.netty.channel.ChannelPipeline
+import io.netty.util.ResourceLeakDetector
+import zio.http.service.{EventLoopGroup, ServerChannelFactory}
+import zio.{ULayer, ZIO, ZLayer}
+import zio.http.service.ServerSSLHandler.ServerSSLOptions
+
+import java.net.{InetAddress, InetSocketAddress}
+
+object ServerConfig {
+
+  val default: ZLayer[Any, Nothing, Config with EventLoopGroup with ServerChannelFactory] = {
```

Comment:
I think this should simply be: `ZLayer[Any, Nothing, ServerConfig]`. 

Then we push the creation of `EventLoopGroup` and `ServerChannelFactory`, based on the `ChannelType`, into `Server` layer, not here. This will be clean and uniform and consistent with ZIO 2 idioms.

Reply:
Done